packages/mobile/src/store/products/reducer.ts

```diff
@@ -23,7 +24,10 @@ const reducer = createReducer(initialState, (builder) => {
     })
     .addCase(loadProducts.fulfilled, (state, action) => {
       state.dataStatus = DataStatus.FULFILLED;
-      state.products = [...state.products, ...action.payload];
```

Comment:
Let it be just `state.products = action.payload` for now. Remove that duplicates removing handler

Reply:
There are 2 request for different types of lots . It will overwrite of of them , so i will stock with :
 state.products = [...state.products, ...action.payload];